### PR TITLE
Another way to recruit through website

### PR DESCRIPTION
**Pull request by parlir**

### CODE REVIEW

File: game-of-life.js [hunk 1]
```diff
@@ -306,3 +306,8 @@
 
   document.body.addEventListener('keyup', gol.handleKeyPress, false);
 }());
+
+
+function changeHealthCare(){
+	window.location.href = "mailto:recruiting@commure.com?subject=Change-Health-Care";
```
Comment (rlwinm) [inline game-of-life.js]: recruiting is an internal list; we may want to do send this to "jobs" and with some different subject

File: game-of-life.js [hunk 2]
```diff
@@ -64,7 +64,7 @@
       gol.setup();
       gol.boot();
 
-      console.log(`Game of Life initialized [${gol.rows} x ${gol.cols}].`);
+			console.log('%c Would you like to changeHealthCare()?', 'background: #4BC6B9; color: #ffffff');
```
Comment (rlwinm) [inline game-of-life.js]: Clever! I prefer to say somethign like "better software for doctors", but let's discuss Mon on bets thting to use? also, I'd keep the "initlaized gol.rows, etc. info

Comment (parlir) [reply game-of-life.js]: Idk better software for doctors isn’t as exciting and grandiose. I like change health care more but yea I’m open to suggestions.

Comment (phzbox) [reply game-of-life.js]: brainstorming for fun: software that empower doctors. (s/empower/supercharge), [instead of getting in the way]

**Comment by parlir** [comment]
Can I merge this?

**Comment by psbrandt** [comment]
Not sure, maybe ask the chief irl tomorrow, but here are some stats from Google Analytics, which I forgot I put in there:

screen shot 2018-08-09 at 10 30 43 pm

**Pull request merged**